src/index.js

```diff
@@ -1,10 +0,0 @@
-import getCoinApplication from './get-coin-application.js'
```

Comment:
Why is this file empty?

Reply:
I got ahead of myself there. It's not empty anymore.